### ISSUE TITLE
attribute-indentation not working fine for block code

### ISSUE THREAD

**Issue by Willibaur** [ember-template-lint/ember-template-lint#509]
Hi there, awesome addon by the way.

I have a case where if I use 

```hbs
{{#sp-ember-select
  errors=customer.errors.title
  label="Title"
as |title|}}
{{title}}
{{/sp-ember-select}}
```
I get an error

> Incorrect indentation of close curly braces '}}' for the component '{{sp-ember-select}}' beginning at 
   L21:C22. Expected '{{sp-ember-select}}' to be at L22:C12

So I do the change as per the docs says to

```hbs
as |title|
}}
```
Then error changes to

> Incorrect indentation of close curly braces '}}' for the component '{{sp-ember-select}}' beginning at 
  L22:C12. Expected '{{sp-ember-select}}' to be at L21:C22

meaning that it was right originally, I think there is a bug there.

I'm using `ember-cli-template-lint": "^1.0.0-beta.2"`

Thank you very much

**Comment by ohcibi** [comment]
I have another example here: https://gist.github.com/ohcibi/783c09f275c901ff0b4396f74542e194

```
node_modules/.bin/ember-template-lint test.hbs
...../template.hbs
  10:2  error  Incorrect indentation of block params '}}' beginning at L10:C2. Expecting the block params to be at L9:C2.  attribute-indentation
  21:15  error  Incorrect indentation of close curly braces '}}' for the component '{{power-calendar}}' beginning at L21:C15. Expected '{{power-calendar}}' to be at L22:C2.  attribute-indentation
```

**Comment by john-griffin** [comment]
I believe this was addressed in `v1.0.0-beta.5`.

**Comment by Willibaur** [comment]
I can confirm that with the following setting

```js
'attribute-indentation': {
  'element-open-end': 'new-line',
  'mustache-open-end': 'new-line',
  'open-invocation-max-len': 120
}
```

everything works fine apart from `'mustache-open-end': 'new-line'` as the problem mentioned originally still persists.

I'm using `ember-cli-template` version `1.0.0-beta.2` which uses `ember-template-lint` version `1.0.5-beta.5`

**Comment by deanpapastrat** [comment]
I'm getting the same error here:

```hbs
    {{#power-select
      selected=newVersion
      options=apiVersions
      onchange=(action (mut newVersion))
      placeholder="Select a version..."
      renderInPlace=true
      searchEnabled=false
    as |version|}}
      {{#if (eq version apiVersions.firstObject)}}
        {{version}} (Latest)
      {{else}}
        {{version}}
      {{/if}}
    {{/power-select}}
```

**Comment by john-griffin** [comment]
Does running from latest `master` work?

**Comment by happydude** [comment]
I'm seeing the same thing with a similar power-select.

I tried npm install https://github.com/ember-template-lint/ember-template-lint.git , ran ember server and got caught in the same loop when I tried to fix it. I killed ember server and restarted and noticed that one of my errors had gone away, so I tried fixing the errors reported by ./node_modules/.bin/ember-template-lint filename.  After I made the needed changes ember-template-lint did not report any errors.

After this I tested breaking my template and reverting it and the loop no longer occurs.

**Comment by ohcibi** [comment]
@Happydude I try to always use the raw linter in such cases to rule out weird addon/plugin/whatever configuration or data flow behavior. I did for this issue and can confirm that the given examples are not fixable with the default configuration even when using the plain linter binary as you did it.

**Comment by Willibaur** [comment]
I agree with @ohcibi linter config is not working as expected, we should not be finding ways around to make it work.

**Comment by raphaelns-developer** [comment]
Same here. How can i disable this rule while this issue it is not fixed?

**Comment by knownasilya** [comment]
Getting the same issue with anglebracket components:

```hbs
    <SelectedActions
      @all={{this.filteredKeys}}
      @selected={{this.selectedKeys}}
      @onDeselect={{action 'toggleSelect' false}}
      @onSelectAll={{action 'toggleSelect' true}}
      as |selected|
    >
```

>   13:4  error  Incorrect indentation of close bracket '>' for the element '<SelectedActions>' beginning at L13:C4. Expected '<SelectedActions>' to be at L12:C4.  attribute-indentation

**Comment by raphaelns-developer** [comment]
@knownasilya Try this:
```hbs
<SelectedActions
  @all={{this.filteredKeys}}
  @selected={{this.selectedKeys}}
  @onDeselect={{action 'toggleSelect' false}}
  @onSelectAll={{action 'toggleSelect' true}}
as |selected|
>
```

**Comment by puwelous** [comment]
@raphaelns-developer unfortunately it doesn't work ☝️ 
```
  <ButtonWithTask
    class={{bem this "revoke" "confirm"}}
    @task={{task this.authorizedApps.confirmRevoke @row}}
    data-test-authorized-apps-table-actions="confirm-revoke"
  as |isRevokeTaskRunning|
  >
    {{#if isRevokeTaskRunning}}
      {{fa-icon "refresh" spin=true}}
    {{/if}}
    {{t "authorized-apps.revoke-confirm"}}
  </ButtonWithTask>
```
